tests/charts/test_scheduler.py

```diff
@@ -674,6 +674,25 @@ def test_persistence_volume_annotations(self):
         )
         assert {"foo": "bar"} == jmespath.search("spec.volumeClaimTemplates[0].metadata.annotations", docs[0])
 
+    @parameterized.expand(
+        "executor",
+        [
+            ("LocalExecutor"),
+            ("LocalKubernetesExecutor"),
+            ("CeleryExecutor"),
+            ("KubernetesExecutor"),
+            ("CeleryKubernetesExecutor"),
+        ],
```

Comment:
```suggestion
        [
            "LocalExecutor",
            "LocalKubernetesExecutor",
            "CeleryExecutor",
            "KubernetesExecutor",
            "CeleryKubernetesExecutor",
        ]
```

This will fix one of them at least.